packages/editor/src/client/panel/authorEditPanel.tsx

```diff
@@ -139,62 +140,74 @@ export function AuthorEditPanel({id, onClose, onSave}: AuthorEditPanelProps) {
       </Drawer.Header>
 
       <Drawer.Body>
-        <Panel>
-          <Form fluid={true}>
-            <FormGroup>
-              <ControlLabel>{t('authors.panels.name')}</ControlLabel>
-              <FormControl
-                name={t('authors.panels.name')}
-                value={name}
-                disabled={isDisabled}
-                onChange={value => {
-                  setName(value)
-                  setSlug(slugify(value))
-                }}
-              />
-            </FormGroup>
-          </Form>
-        </Panel>
-
-        <ChooseEditImage
-          image={image}
-          disabled={isLoading}
-          openChooseModalOpen={() => setChooseModalOpen(true)}
-          openEditModalOpen={() => setEditModalOpen(true)}
-          removeImage={() => setImage(undefined)}
-        />
-
-        <Panel header={t('authors.panels.links')}>
-          <ListInput
-            value={links}
-            onChange={links => setLinks(links)}
-            defaultValue={{title: '', url: ''}}>
-            {({value, onChange}) => (
-              <div style={{display: 'flex', flexDirection: 'row'}}>
-                <Input
-                  placeholder={t('authors.panels.title')}
-                  style={{
-                    flexBasis: '30%',
-                    marginRight: '10px'
-                  }}
-                  value={value.title}
-                  onChange={title => onChange({...value, title})}
-                />
-                <Input
-                  placeholder={t('authors.panels.link')}
-                  style={{
-                    flexBasis: '70%'
+        <PanelGroup accordion>
+          <Panel header={t('authors.panels.name')} defaultExpanded>
+            <Form fluid={true}>
+              <InputGroup inside>
+                <InputGroup.Addon>
+                  <Icon icon="avatar" />
+                </InputGroup.Addon>
+                <FormControl
+                  name={t('authors.panels.name')}
+                  value={name}
+                  disabled={isDisabled}
+                  onChange={value => {
+                    setName(value)
+                    setSlug(slugify(value))
                   }}
-                  value={value.url}
-                  onChange={url => onChange({...value, url})}
                 />
-              </div>
-            )}
-          </ListInput>
-        </Panel>
-        <Panel header={t('authors.panels.bioInformation')}>
-          <RichTextBlock value={bio} onChange={value => setBio(value)} />
-        </Panel>
+              </InputGroup>
+            </Form>
+          </Panel>
+          <Panel header={t('authors.panels.image')}>
+            <ChooseEditImage
+              image={image}
+              header={''}
+              top={0}
```

Comment:
@0815pascal  Great work, just asking about the usage of top and left values? why not all the sides?

Reply:
`top` and `left` are here used to position the wrench icon. Because its position is set relative to its parent element, it's sufficient if we only indicate `left` and `top`